<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nothing Shop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QScrollArea" name="merchList">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>620</width>
         <height>419</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="merchBox1">
          <property name="title">
           <string>Merch Name</string>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>581</width>
             <height>152</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,8,1">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="merchImage1">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>150</height>
               </size>
              </property>
              <property name="text">
               <string>Image</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="nameLabel1">
                <property name="text">
                 <string>Merch Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextBrowser" name="merchDesc1">
                <property name="placeholderText">
                 <string>Merch description</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="RemainingLabel1">
                <property name="text">
                 <string>Remaining: 0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QSpinBox" name="merchNum">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="BuyButton1">
                  <property name="text">
                   <string>Buy</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="separator"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menuQuit">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuQuit"/>
   <addaction name="menuUser"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
